modules/TodoAppPage/components/CreateTask/CreateTask.tsx

```diff
@@ -3,33 +3,23 @@ import { Button, Modal, Select, TextInput, Textarea } from "@mantine/core";
 import { DateInput } from "@mantine/dates";
 import "@mantine/dates/styles.css";
 
-import {
-  addTaskToLocalStorage,
-  getLastTaskId,
-  getSortedTasks,
-  saveLastTaskId,
-} from "@/shared/utils/localStorage";
+import { getLastTaskId } from "@/shared/utils/localStorage";
 import { TTask } from "@/shared/typedefs/types";
 import { EPriority, ETaskStatus } from "@/shared/typedefs/enums";
+import { useTasksContext } from "@/shared/utils/TasksProvider/TasksProvider";
 
 import { TCreateTaskProps } from "./CreateTask.types";
 
-function CreateTask({
-  opened,
-  close,
-  setTasks,
-  setHistory,
-  history,
-  setCurrStateIndex,
-  currStateIndex,
-}: TCreateTaskProps) {
+function CreateTask({ opened, close }: TCreateTaskProps) {
   const {
     control,
     handleSubmit,
     formState: { errors, isValid },
     reset,
   } = useForm<TTask>({ mode: "onBlur" });
 
+  const { addTask } = useTasksContext();
```

Comment:
Why are we making these changes here? Isn't the scope of the PR to add taskDetails page?